charts/chart-museum/templates/service.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.global.chartMuseumEnabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: helm-museum
+spec:
+  type: ClusterIP
+  selector:
+    app: helm-museum
+  ports:
+  - port: 8080
+    targetPort: 8080
+{{- end }}
```

Comment:
could have an ingress, too, that'd probably make things easier. Don't know how much of a lift that'd be, to update other things to include it. It'd only be accessible on opt-in and via vpn or whatever the customer has configured
```
https://helm.BASEDOMAIN
```

Reply:
We might as well reuse the existing ingress that hosts everything else like grafana, kibana, etc..